interview-questions.md

```diff
@@ -8,36 +8,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Sounds like it's time for a new migration! Blanking off the pseudo of the top of my head, but you'd do something like [rails generate migration add_foreign_key], and then go into the migration file and put whatever code is necessarry to add your foreign key there. as for the name and where it is, I THINK it should be [student_id], in the cohort model.
 
-Researched answer:
+Researched answer: commands should be [rails generate migration add_foreign_key_to_students cohort:references] to create the migration file, [def change
+  add_foreign_key :students, :cohorts
+end
+], inside the migration file to do the adding, and then raild db:migrate back in the console to update the database! Oh, and I got the other two questions backwards. it should be [cohort_id], in the student model. ***.tnemom gniniarb sdrawkcab a dah***
```

Comment:
Nice job following up with the foreign key being in the student model! ***sneppah ti***